src/integration/java/com/smartcar/sdk/VehicleTest.java

```diff
@@ -45,13 +45,23 @@ public void testVin() throws SmartcarException {
     }
 
     /**
-     * Tests that the application permissions can be obtained.
+     * Tests that the vehicle permissions can be obtained.
      */
     @Test(groups = "vehicle")
     public void testPermissions() throws SmartcarException {
         String[] permissions = this.vehicle.permissions();
     }
 
+    /**
+     * Tests that the vehicle has certain permissions.
+     */
+    @Test(groups = "vehicle")
+    public void testHasPermissions() throws SmartcarException {
+        boolean hasPermission = this.vehicle.hasPermissions("read_odometer");
+        boolean hasPermissions = this.vehicle.hasPermissions(new String[]
+                {"read_odometer", "read_location"});
```

Comment:
can we assert that the 2 returns are true?
```
Assert.assertTrue(hasPermission);
Assert.assertTrue(hasPermissions);
```
Reasoning. The other tests in the file are pretty old and we only wrote them to have something, but as we add more we want to improve the tests.

Reply:
added!